src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Square extends FigureTwo implements AreaOfFigure,InfoFigure {
+    //квадрат
+
+    public Square(String color, double firsSide) {
+        super(color, firsSide, firsSide);
+    }
+
+    @Override
+    public double detAreaOfFigure() {
+        return getSecondSide() * getFirsSide();
+    }
+
+    @Override
+    public String infoFigure() {
+        return ("Длина : " + getFirsSide() + " см"
+                + " Ширина : " + getSecondSide() + " см"
+                + " Площадь квадрата = " + detAreaOfFigure());
+    }
+
+    @Override
+    public String infoFromRigure() {
```

Comment:
The method name `infoFromRigure` seems to be a typo. It should likely be `infoFromFigure` to match the naming convention and avoid confusion.